src/aws/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM busybox as mediainfo
+
+# Embed the mediainfo layer in the image
+RUN wget https://mediaarea.net/download/binary/mediainfo/20.09/MediaInfo_CLI_20.09_Lambda.zip && \
+    unzip MediaInfo_CLI_20.09_Lambda.zip -d /tmp/mediainfo
```

Comment:
What about making the version number a build argument with an appropriate default?